src/clr-angular/progress/progress-bars/progress-bar.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'clr-progress-bar',
+  template: `
+  <ng-container>
+    <progress [attr.max]="max" [attr.value]="value" [attr.data-displayval]="displayValue"></progress>
+    <span *ngIf="displayAriaLive()" [attr.aria-live]="liveRegion">{{ displayValue }}</span>
+  </ng-container>
+  `,
+  host: {
+    '[attr.class]': 'elementClass',
+  },
+})
+export class ClrProgressBar {
+  @Input('class') classes: string;
+  @Input('max') max: number = 100;
+  @Input('value') value: number = 0;
+  @Input('displayval') displayval: string;
+  @Input('liveRegion') liveRegion: string = 'polite';
```

Comment:
The Progress bar will need to support taking in an `id` input to set on the inner `id` of the progress bar for a11y. 
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/progress

You can set the inner id of the progress via an input and then delete the outer wrapper id to prevent duplicate ids with a HostBinding. (There might be a better way to do this but this is what I came up with) https://coryrylan.com/blog/prevent-attribute-reflection-in-angular

Reply:
Done ;)